src/utils.js

```diff
@@ -14,20 +15,24 @@ const altWidgetMap = {
   boolean: {
     radio: RadioWidget,
     select: SelectWidget,
+    hidden: HiddenWidget,
   },
   string: {
     password: PasswordWidget,
     radio: RadioWidget,
     select: SelectWidget,
     textarea: TextareaWidget,
+    hidden: HiddenWidget,
   },
   number: {
     updown: UpDownWidget,
     range: RangeWidget,
+    hidden: HiddenWidget,
   },
   integer: {
     updown: UpDownWidget,
     range: RangeWidget,
+    hidden: HiddenWidget,
   }
 };
```

Comment:
We should add the "date-time" jsonschema type in here so we can support hiding related widget as well.